python/parser.js

```diff
@@ -92,6 +93,10 @@
 function layoutRegions(arr){
     return arr.reduce((acc, val) => `${acc}.layout(${regionParameter(val)})`, '')
 }
+function layoutBreakpoints(arr){
+    if (Array.isArray(arr)) return `.layout_breakpoints(` + arr.join(',') + `)`
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8717150)
```suggestion
    if (Array.isArray(arr)) return ".layout_breakpoints(" + arr.join(',') + `)`
```